capivaraProjects/src/validate/validateServer.tsx

```diff
@@ -0,0 +1,25 @@
+import client from '../apolloConfig/apollo';
+import { gql } from 'apollo-boost';
+import { Alert } from 'react-native';
+import { validate } from 'json-schema';
+
+function validateServer(email: string, password : string){
+
+    // Eu ia usar variaveis auxiliares($email, $password), mas n encontrei a documentacao utilizando nesse formato, apenas useQuery.
+    return (
+        client.mutate({
+            mutation: gql`
+                mutation login {
+                    login(data : {email : "${email}", password : "${password}"}) {
+                    token
+                    user {
+                        id
+                    }
+                    }
+                }
```

Comment:
Poderia ter usado os parâmetros do graphql
O tipo `LoginInputType` é declarado no server e voce pode conferir ele no [playground](https://tq-template-server-sample.herokuapp.com/) clicando na aba verde a direita (Schema)
```suggestion
                mutation login($data: LoginInputType!) {
                    login(data : $data) {
                    token
                    user {
                        id
                    }
                    }
                }
```